moztelemetry/event-format/run.sh

```diff
@@ -0,0 +1,53 @@
+array_event_file="events_array.json"
+obj_event_file="events_obj.json"
+
+tee $array_event_file <<EOF
+[
+    [194010,"addonsManager","uninstall-other-gmp-widevinecdm",{"source":"gmp-plugin"}],
```

Comment:
I plucked these from the events table, and joined the object-method-value for the event_name.